modules/_UXTeam.scss

```diff
@@ -0,0 +1,104 @@
+
+#UXTeam{
+    text-align: center;
+
+.tittle{
+        padding: 2rem 0;
```

Comment:
Be consistent in your code - use either units like rem or your variables for sizes